app/routes/pages/components/PageDetail.js

```diff
@@ -91,9 +91,9 @@ function PageDetail<T: Object>({
     <Content>
       <NavigationTab title={title}>
         {actionGrant.includes('edit') &&
-          editUrl && <NavigationLink to={`${editUrl}`}>ENDRE</NavigationLink>}
+          editUrl && <NavigationLink to={`${editUrl}`}>Endre</NavigationLink>}
```

Comment:
This is kinda the opposite problem – the purpose of the ```{`someString${someVar}`}``` syntax is to combine strings and variables. My earlier comments tell you to use only "" when all you have is strings, but here you only have a variable. Thus, you can just write `to={editUrl}`.